pimd/pim6_cmd.c

```diff
@@ -1569,6 +1569,224 @@ DEFPY (show_ipv6_pim_nexthop_lookup,
 	return CMD_SUCCESS;
 }
 
+DEFPY (show_ipv6_mroute,
+       show_ipv6_mroute_cmd,
+       "show ipv6 mroute [vrf NAME] [X:X::X:X$s_or_g [X:X::X:X$g]] [fill$fill] [json$json]",
+       SHOW_STR
+       IPV6_STR
+       MROUTE_STR
+       VRF_CMD_HELP_STR
+       "The Source or Group\n"
+       "The Group\n"
+       "Fill in Assumed data\n"
+       JSON_STR)
+{
+	pim_sgaddr sg = {0};
+	struct pim_instance *pim;
+	struct vrf *v;
+	json_object *json_parent = NULL;
+
+	v = vrf_lookup_by_name(vrf ? vrf : VRF_DEFAULT_NAME);
```

Comment:
`v` can be NULL if the user specifies a vrf name that does not exist, i.e. `vrf != NULL` but no such named VRF exists.

(issue in multiple places)

Reply:
Fixed